telegram/_chat.py

```diff
@@ -388,10 +388,24 @@ def __init__(
 
         self._freeze()
 
+    @property
+    def effective_name(self) -> Optional[str]:
+        """
+        :obj:`str`: Convenience property. Gives :attr:`title` if not :obj:`None`,
+        else :attr:`full_name` if not :obj:`None`.
+
+        .. versionadded:: 20.1
+        """
+        if self.title is not None:
+            return self.title
+        if self.full_name is not None:
+            return self.full_name
+        return None
```

Comment:
You are not testing this line, please add on as:
```
chat = Chat(id=1, type=Chat.GROUP)
assert chat.effective_name is None

```

Reply:
Yes saw that, I'm on something else right now but this will be done in ~30mins